ciphers/base64_encoding.py

```diff
@@ -71,7 +81,27 @@ def base64_decode(encoded_data: str) -> bytes:
     True
     >>> base64_decode(c) == b64decode(c)
     True
+    >>> base64_decode("abc")
+    Traceback (most recent call last):
+      ...
+    AssertionError: Incorrect padding
     """
+    # Make sure encoded_data is either a string or a bytes-like object
+    if not isinstance(encoded_data, bytes) and not isinstance(encoded_data, str):
+        raise TypeError(
+            "argument should be a bytes-like object or ASCII string, not '{}'".format(
+                encoded_data.__class__.__name__
```

Comment:
```suggestion
            f"argument should be a bytes-like object or ASCII string, not '{encoded_data.__class__.__name__}'"
```

Reply:
I was using f-strings at the beginning, but doing so exceeds the 88 chars per line in this case, and flake8 reports an error.
What should I do?